### ISSUE TITLE
When i change workspaces, the window i am focused is freezing.

### ISSUE THREAD

**Issue by Gorkido** [fuhsjr00/bug.n#282]
https://user-images.githubusercontent.com/81165187/124512393-ec315180-dde0-11eb-9165-2a768e39904c.mp4

Config: 

; bug.n - tiling window management
; @version 9.0.0

;; Status bar
Config_showBar=1
Config_horizontalBarPos=center
Config_verticalBarPos=tray
Config_barWidth=25%
Config_singleRowBar=1
Config_spaciousBar=0
Config_fontName=DejaVu Sans Mono
Config_fontSize=8
Config_largeFontSize=24
Config_backColor_#1=101010;603000;Black;603000;Black;603000;000060;404040;080808
Config_backColor_#2=808080;;;;;;;202020;080808
Config_foreColor_#1=404040;101010;Black;101010;Black;101010;Black;404040;202020
Config_foreColor_#2=4040ff;;;;;;;0000ff;0000ff
Config_foreColor_#3=;;;;;;;010101;
Config_fontColor_#1=White;White;White;White;White;White;White;Black;808080
Config_fontColor_#2=Black;;;;;;;White;White
Config_fontColor_#3=;;;;;;;Black;
Config_barTransparency=off
Config_barCommands=Run, explore doc;Monitor_toggleBar();Main_reload();Reload;ExitApp
Config_readinBat=0
Config_readinCpu=0
Config_readinDate=0
Config_readinDiskLoad=0
Config_readinMemoryUsage=0
Config_readinNetworkLoad=0
Config_readinTime=0
Config_readinTimeFormat=HH:mm
Config_readinVolume=0
;Config_readinInterval=5000

;; Windows ui elements
;Config_bbCompatibility=0
;Config_borderWidth=0
;Config_borderPadding=10
Config_showTaskBar=1
;Config_showBorder=1
;Config_selBorderColor=

;; Window arrangement
Config_viewNames=1;2;3;4;5;6;7;8;9
Config_layout_#1=[]=;tile
Config_layout_#2=[M];monocle
Config_layout_#3=><>;
Config_layoutCount=3
Config_layoutAxis_#1=1
Config_layoutAxis_#2=2
Config_layoutAxis_#3=2
Config_layoutGapWidth=50
Config_layoutMFactor=0.6
Config_areaTraceTimeout=1000
Config_continuouslyTraceAreas=0
Config_dynamicTiling=1
Config_ghostWndSubString= (Not Responding)
Config_mFactCallInterval=700
Config_mouseFollowsFocus=1
Config_newWndPosition=top
Config_onActiveHiddenWnds=view
Config_shellMsgDelay=350
Config_syncMonitorViews=0
Config_viewFollowsTagged=0
Config_viewMargins=0;0;0;0

;; Config_rule_#<i> := '<class>;<title>;<function name>;<is managed>;<m>;<tags>;<is floating>;<is decorated>;<hide title>;<action>'
;Config_rule_#1=.*;.*;;1;0;0;0;0;0;
;Config_rule_#2=.*;.*;Window_isChild;0;0;0;1;1;1;
;Config_rule_#3=.*;.*;Window_isPopup;0;0;0;1;1;1;
;Config_rule_#4=QWidget;.*;;1;0;0;0;0;0;
;Config_rule_#5=SWT_Window0;.*;;1;0;0;0;0;0;
;Config_rule_#6=Xming;.*;;1;0;0;0;0;0;
;Config_rule_#7=MsiDialog(No)?CloseClass;.*;;1;0;0;1;1;0;
;Config_rule_#8=AdobeFlashPlayerInstaller;.*;;1;0;0;1;0;0;
;Config_rule_#9=CalcFrame;.*;;1;0;0;1;1;0;
;Config_rule_#10=CabinetWClass;.*;;1;0;0;0;1;0;
;Config_rule_#11=OperationStatusWindow;.*;;0;0;0;1;1;0;
;Config_rule_#12=Chrome_WidgetWin_1;.*;;1;0;0;0;1;0;
;Config_rule_#13=Chrome_WidgetWin_1;.*;Window_isPopup;0;0;0;1;1;1;
;Config_rule_#14=Chrome_RenderWidgetHostHWND;.*;;0;0;0;1;1;0;
;Config_rule_#15=IEFrame;.*Internet Explorer;;1;0;0;0;1;0;
;Config_rule_#16=MozillaWindowClass;.*Mozilla Firefox;;1;0;0;0;1;0;
;Config_rule_#17=MozillaDialogClass;.*;;1;0;0;1;1;0;
;Config_rule_#18=ApplicationFrameWindow;.*Edge;;1;0;0;0;1;0;

;; Configuration management
;Config_autoSaveSession=auto
;Config_maintenanceInterval=5000

;; Key definitions
;; Window management
;Config_hotkey=#Down::View_activateWindow(0, +1)
;Config_hotkey=#Up::View_activateWindow(0, -1)
;Config_hotkey=#+Down::View_shuffleWindow(0, +1)
;Config_hotkey=#+Up::View_shuffleWindow(0, -1)
;Config_hotkey=#+Enter::View_shuffleWindow(1)
;Config_hotkey=#c::Manager_closeWindow()
;Config_hotkey=#+d::Window_toggleDecor()
;Config_hotkey=#+f::View_toggleFloatingWindow()
;Config_hotkey=#+m::Manager_moveWindow()
;Config_hotkey=#+s::Manager_sizeWindow()
;Config_hotkey=#+x::Manager_maximizeWindow()
;Config_hotkey=#i::Manager_getWindowInfo()
;Config_hotkey=#+i::Manager_getWindowList()
;Config_hotkey=!Down::View_moveWindow(0, +1)
;Config_hotkey=!Up::View_moveWindow(0, -1)
;Config_hotkey=!+Enter::Manager_maximizeWindow()
;Config_hotkey=!1::View_moveWindow(1)
;Config_hotkey=!2::View_moveWindow(2)
;Config_hotkey=!3::View_moveWindow(3)
;Config_hotkey=!4::View_moveWindow(4)
;Config_hotkey=!5::View_moveWindow(5)
;Config_hotkey=!6::View_moveWindow(6)
;Config_hotkey=!7::View_moveWindow(7)
;Config_hotkey=!8::View_moveWindow(8)
;Config_hotkey=!9::View_moveWindow(9)
;Config_hotkey=!0::View_moveWindow(10)
;Config_hotkey=!BackSpace::View_toggleStackArea()

;; Window debugging
;Config_hotkey=#^i::Debug_logViewWindowList()
;Config_hotkey=#+^i::Debug_logManagedWindowList()
;Config_hotkey=#^h::Debug_logHelp()
;Config_hotkey=#^d::Debug_setLogLevel(0, -1)
;Config_hotkey=#^+d::Debug_setLogLevel(0, +1)

;; Layout management
;Config_hotkey=#Tab::View_setLayout(-1)
;Config_hotkey=#f::View_setLayout(3)
;Config_hotkey=#m::View_setLayout(2)
;Config_hotkey=#t::View_setLayout(1)
;Config_hotkey=#Left::View_setLayoutProperty(MFactor, 0, -0.05)
;Config_hotkey=#Right::View_setLayoutProperty(MFactor, 0, +0.05)
;Config_hotkey=#^t::View_setLayoutProperty(Axis, 0, +1, 1)
;Config_hotkey=#^Enter::View_setLayoutProperty(Axis, 0, +2, 1)
;Config_hotkey=#^Tab::View_setLayoutProperty(Axis, 0, +1, 2)
;Config_hotkey=#^+Tab::View_setLayoutProperty(Axis, 0, +1, 3)
;Config_hotkey=#^Up::View_setLayoutProperty(MY, 0, +1)
;Config_hotkey=#^Down::View_setLayoutProperty(MY, 0, -1)
;Config_hotkey=#^Right::View_setLayoutProperty(MX, 0, +1)
;Config_hotkey=#^Left::View_setLayoutProperty(MX, 0, -1)
;Config_hotkey=#+Left::View_setLayoutProperty(GapWidth, 0, -2)
;Config_hotkey=#+Right::View_setLayoutProperty(GapWidth, 0, +2)

;; View/Tag management
;Config_hotkey=#+n::View_toggleMargins()
;Config_hotkey=#BackSpace::Monitor_activateView(-1)
;Config_hotkey=#+0::Monitor_setWindowTag(10)
;Config_hotkey=#1::Monitor_activateView(1)
;Config_hotkey=#+1::Monitor_setWindowTag(1)
;Config_hotkey=#^1::Monitor_toggleWindowTag(1)
;Config_hotkey=#2::Monitor_activateView(2)
;Config_hotkey=#+2::Monitor_setWindowTag(2)
;Config_hotkey=#^2::Monitor_toggleWindowTag(2)
;Config_hotkey=#3::Monitor_activateView(3)
;Config_hotkey=#+3::Monitor_setWindowTag(3)
;Config_hotkey=#^3::Monitor_toggleWindowTag(3)
;Config_hotkey=#4::Monitor_activateView(4)
;Config_hotkey=#+4::Monitor_setWindowTag(4)
;Config_hotkey=#^4::Monitor_toggleWindowTag(4)
;Config_hotkey=#5::Monitor_activateView(5)
;Config_hotkey=#+5::Monitor_setWindowTag(5)
;Config_hotkey=#^5::Monitor_toggleWindowTag(5)
;Config_hotkey=#6::Monitor_activateView(6)
;Config_hotkey=#+6::Monitor_setWindowTag(6)
;Config_hotkey=#^6::Monitor_toggleWindowTag(6)
;Config_hotkey=#7::Monitor_activateView(7)
;Config_hotkey=#+7::Monitor_setWindowTag(7)
;Config_hotkey=#^7::Monitor_toggleWindowTag(7)
;Config_hotkey=#8::Monitor_activateView(8)
;Config_hotkey=#+8::Monitor_setWindowTag(8)
;Config_hotkey=#^8::Monitor_toggleWindowTag(8)
;Config_hotkey=#9::Monitor_activateView(9)
;Config_hotkey=#+9::Monitor_setWindowTag(9)
;Config_hotkey=#^9::Monitor_toggleWindowTag(9)

;; Monitor management
;Config_hotkey=#.::Manager_activateMonitor(0, +1)
;Config_hotkey=#,::Manager_activateMonitor(0, -1)
;Config_hotkey=#+.::Manager_setWindowMonitor(0, +1)
;Config_hotkey=#+,::Manager_setWindowMonitor(0, -1)
;Config_hotkey=#^+.::Manager_setViewMonitor(0, +1)
;Config_hotkey=#^+,::Manager_setViewMonitor(0, -1)

;; GUI management
;Config_hotkey=#+Space::Monitor_toggleBar()
;Config_hotkey=#Space::Monitor_toggleTaskBar()
;Config_hotkey=#y::Bar_toggleCommandGui()
;Config_hotkey=#+y::Monitor_toggleNotifyIconOverflowWindow()
;Config_hotkey=!+y::View_traceAreas()

;; Administration
;Config_hotkey=#^e::Run, edit Config.ini
;Config_hotkey=#^s::Config_UI_saveSession()
;Config_hotkey=#^+r::Main_relod()
Config_hotkey=#^+r::Reload
Config_hotkey=#^q::ExitApp

**Comment by KSR-Yasuda** [comment]
Is this Chrome?
If so, I am experiencing the same issue.

I noticed this since late Oct-2021, Chrome v95 or later (on Win 10 Pro x64) and bug.n v9.0.2.

Chrome v95 once had some rendering issue on Android:
some pages had got white out after page load.

Rendering problem of Chrome Android was silently fixed (probably Google's setting?),
however, this Chrome and bug.n rendering issue has came out then.

I doubt that Chrome v95 or later has changed some rendering method or something to be uncooperative with bug.n
(As Android had once gone wrong, it would have been fixed something in page rendering).

So I think this is rather Chrome's problem, though, there's no sign of bugfix...
